source/virtualBuffers/__init__.py

```diff
@@ -334,6 +342,10 @@ def _normalizeControlField(self,attrs):
 		return attrs
 
 	def _normalizeFormatField(self, attrs):
+		strippedCharsFromStart = attrs.get("strippedCharsFromStart")
+		if strippedCharsFromStart is not None:
+			assert strippedCharsFromStart.isdigit, "strippedCharsFromStart is no digit, %r" % strippedCharsFromStart

```

Comment:
1. That should be `isdigit()` (method call), not just `isdigit` (attribute). `isdigit` will always be True. :)
2. nit: "is no digit" -> "is not a digit"